src/modules/uORB/uORBDeviceMaster.cpp

```diff
@@ -311,8 +313,6 @@ void uORB::DeviceMaster::showTop(char **topic_filter, int num_filters)
 		}
 	}
 
-	PX4_INFO_RAW("\033[2J\n"); //clear screen
```

Comment:
Removing this leads to this after the first output:
```
update: 1s, num topics: 78
TOPIC NAME      INST #SUB RATE #LOST #Q SIZE
sensor_gyro_fifo   0    0  401     0  1  224
sensor_accel_fifo  0    0  401     0  1  224
FW git-branch: pr-uorb_top_minor
OS: NuttX
OS version: Release 8.2.0 (134349055)
OS git-hash: 66b4f2c4f2128994f8e8a908d4888f6d37565cfd
Build datetime: May  6 2020 08:46:02
Build uri: localhost
Toolchain: GNU GCC, 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
PX4GUID: 0002000000003236343532375118002b0024
MCU: STM32F76xxx, rev. Z
[hardfault_log] Fault Log is Armed
INFO  [param] selected parameter default file /fs/mtd_params
INFO  [tune_control] Publishing standard tune 1
Board defaults: /etc/init.d/rc.board_defaults
Board sensors: /etc/init.d/rc.board_sensors
icm20689 #0 on SPI bus 1 (devid=0x3c)
bmi055 #0 on SPI bus 1 (devid=0x41)
bmi055 #1 on SPI bus 1 (devid=0x42)
nsh: icm20948: command not found
ist8310 #0 on I2C bus 3
ms5611 #0 on SPI bus 4 (devid=0x3d)
Board extras: /etc/init.d/rc.board_mavlink
INFO  [mavlink] mode: Config, data rate: 800000 B/s on /dev/ttyACM0 @ 57600B
Starting Main GPS on /dev/ttyS0
Starting MAVLink on /dev/ttyS1
INFO  [mavlink] mode: Normal, data rate: 1200 B/s on /dev/ttyS1 @ 57600B
INFO  [px4io] default PWM output device
INFO  [init] Mixer: /etc/mixers/vtol_convergence.main.mix on /dev/pwm_output0
INFO  [airspeed_selector] No airspeed sensor detected. Switch to non-airspeed mode.
WARN  [airspeed_selector] Airspeed: switched from sensor 0 to -1
INFO  [logger] logger started (mode=all)

NuttShell (NSH)
nsh> INFO  [ecl/EKF] reset position to last known position
INFO  [ecl/EKF] reset velocity to zero

nsh> uorb top fifo
```

Reply:
I had removed that along the way trying to find what was killing this on the test rack (pyserial), but it now looks like removing the read from stdin made the difference.

Initial clear readded.